src/blockchain_utilities/blockchain_blackball.cpp

```diff
@@ -525,6 +526,7 @@ int main(int argc, char* argv[])
           state.spent.insert(output_data(od.amount, last_unknown));
         }
       }
+      work_spent.erase(work_spent.begin());
```

Comment:
Erase elements instead of simply looping through?

Reply:
Either would do. I'd started doing something else, which required this, but it's now not needed, I'll change it back.